libtiledbsoma/cmake/Superbuild.cmake

```diff
@@ -63,8 +63,8 @@ set(INHERITED_CMAKE_ARGS
   -DTILEDB_SERIALIZATION=${TILEDB_SERIALIZATION}
   -DTILEDB_WERROR=${TILEDB_WERROR}
   -DTILEDB_VERBOSE=${TILEDB_VERBOSE}
+  -DTILEDB_SANITIZER=${TILEDB_SANITIZER}
```

Comment:
```suggestion
```

We don't need this with the multi-config arrangement, right?

Reply:
Enabling the sanitizer on TileDB embedded is a separate option.

Reply:
Oh I forgot about that.